lib/plausible_api/configuration.rb

```diff
@@ -0,0 +1,9 @@
+module PlausibleApi
+  class Configuration
+    attr_accessor :base_url
+
+    def base_url
+      @base_url || "https://plausible.io"
```

Comment:
What I often do is setup defaults in initialize and then do an attr_accessor as you have for overriding. Then you aren't making the same string over and over. Minor so every one feel free to ignore me.

Reply:
updated. let me know if that's what you meant.